### ISSUE TITLE
 Belted Kingfisher diet data missing

### ISSUE THREAD

**Issue by joewcorra** [hurlbertlab/dietdatabase#164]
Belted Kingfisher appears in NA_specieslist but I do not see diet data in either AvianDietDatabase.txt or the dietdb object. The species account in Birds of the World references studies including quantitative diet data; there's also data in more recent ecotoxicology work like [Seston, et al. (2012)](https://setac.onlinelibrary.wiley.com/doi/full/10.1002/etc.1780).

**Comment by ahhurlbert** [comment]
You are correct that there are no Belted Kingfisher diet data in the database as of yet. (NA_specieslist is not a list of species with data)  Thanks for pointing to some references for us to include, possibly this summer or fall.

**Comment by elpin3** [comment]
- [ ] [Belted Kingfisher Stomach Contents ~1930](https://backend.production.deepblue-documents.lib.umich.edu/server/api/core/bitstreams/22c894b4-f842-4432-8691-d2dc97b11853/content)
- [ ] [Food Preferences and Feeding Habits of the Pied Kingfisher Ceryle rudis ](https://www.jstor.org/stable/3676115?seq=3)
- [ ] [Observations on the Breeding Biology and Behavior of a Nesting Population of Belted Kingfishers](https://digitalcommons.usf.edu/cgi/viewcontent.cgi?article=9400&context=condor)
- [ ] [DIET OF VERTEBRATE PREDATORS ON TROUT WATERS IN NORTH CENTRAL LOWER MICHIGAN](https://www.dnr.state.mi.us/publications/pdfs/DNRFishLibrary/ResearchReports/RR1801-RR1900/RR1839.pdf)
- [ ] [Predatory efficiency and energetics of belted kingfishers wintering along the Mad River - Table 4 Page 28](https://scholarworks.calstate.edu/concern/theses/0c483m62h)
